pkg/liquidity-mining/contracts/VeBoostV2.vy

```diff
@@ -225,39 +271,15 @@ def _boost(_from: address, _to: address, _amount: uint256, _endtime: uint256):
 
 @external
 def boost(_to: address, _amount: uint256, _endtime: uint256, _from: address = msg.sender):
-    # reduce approval if necessary
-    if _from != msg.sender:
+    if _from != msg.sender and self.isApprovedForAll[_from][msg.sender] == False:
```

Comment:
If we just set the allowance from delegator to operator to `MAX_UINT256` we shouldn't need anything related to `isApprovedForAll`. Am I missing anything?

Reply:
No; as above, was just done for consistency with the previous preseed contract, and self-documentation. Otherwise there's no indication it was done (so I can just document it here instead).